scripts/multiple_OEP_registration.py

```diff
@@ -0,0 +1,26 @@
+import os
+import pathlib
+
+import requests
+from dotenv import load_dotenv
+
+
+load_dotenv()
+
+
+path = pathlib.Path("Path to metadata")
```

Comment:
If you write it like this, user has to change script and this could lead to unwished git commits, changing the script.
My suggestions is to put everything into a function and make `path` a parameter to the function.